index.html

```diff
@@ -14,15 +14,18 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">Leslie Gudo</a>
-            <ul class="navbar-list">
-
-            <li class="menu-icon">
-                <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </li>
-            <li class="desktop-nav-list">Portfolio</li>
-            <li class="desktop-nav-list">About</li>
-            <li class="desktop-nav-list">Contact</li>
+            <div id="modal" class="modal hidden">
+              <span id="modalClose" class="modalClose">&times;</span>
+            <ul id="navLinks" class="navbar-list">
+            <li class="desktop-nav-list"><a href="#">Portfolio</a></li>
+            <li class="desktop-nav-list"><a href="#about">About</a></li>
+            <li class="desktop-nav-list"><a href="#contact">Contact</a></li>
```

Comment:
- [ ] When I'm viewing on the Mobile screen and click on a menu item, the mobile menu is not closing. What happens is, as I click on either about or contact, the page scrolls to that section, and the text stays blurred due to the styles passed on the open menu. The menu does not close when an item is clicked. Kindly fix that so that when an item in the menu list is clicked, the mobile menu closes and the user is taken to that section. 👍🏽

**See how the texts look blurred due to the mobile many still opens even when a link is clicked**

<img width="530" alt="Screenshot 2023-01-31 at 11 39 43 AM" src="https://user-images.githubusercontent.com/50941074/215750426-ae380660-dcbc-43fb-b9e1-f6f834483add.png">

- [ ] Also, the mobile menu is not fixed. It's supposed to stay fixed and no scrolling is possible. Only when an item is clicked, does it then close and takes the user to that section. Kindly fix this 👍🏽

<img width="468" alt="Screenshot 2023-01-31 at 11 33 02 AM" src="https://user-images.githubusercontent.com/50941074/215749095-be3cd3ae-9942-41e9-a800-c6310be5d510.png">